packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -88,28 +88,23 @@ async function getAndUpdateSidechainConverterKeys(): Promise<string> {
   if (!keysTableItems.Items) {
     throw new Error(`No keys found in sidechain keys table`);
   }
-  let activeKey: string | undefined = undefined;
+  const activeKeys: string[] = [];
```

Comment:
We talked about this last time and now that I look at it again I’m even more convinced we should modify this.

We’re reviewing all keys on every conversion. We could just return the first key we find and let the next execution do the same.

We don't get any benefit from reviewing all keys, since that's done every time - so we might as well just return from the first key we find.

Reply:
there's like 350 array items, even if there was 10k this wouldn't be close to an expensive operation

normally your suggestion is a no-brainer - but this is a temporary hack, and the main benefit of this right now is it allows us to pick a key at random, instead if sequentially exhausting keys

Reply:
yeah, good point about pick one at random instead of sequentially exhausting them